AUTHORS

```diff
@@ -11,6 +11,7 @@ Patches and Suggestions
 
 - Adam Zapletal
 - Ali Afshar
+- Andrii Soldatenko
```

Comment:
@ThiefMaster Sorry for that. Will remove myself.
Also I'll try to fix unit tests on Python 3.6